src/biom_subsampled.cpp

```diff
@@ -0,0 +1,169 @@
+/*
+ * BSD 3-Clause License
+ *
+ * Copyright (c) 2023-2023, UniFrac development team.
+ * All rights reserved.
+ *
+ * See LICENSE file for more details
+ */
+
+#include <cstdlib>
+#include <iostream>
+#include <stdio.h>
+#include <random>
+
+#include "biom_subsampled.hpp"
+
+using namespace su;
+
+linked_sparse_transposed::linked_sparse_transposed(sparse_data &other) 
```

Comment:
To verify, this is transposing the sparse compressed matrix?

Reply:
Yes.